src/content/developers/docs/smart-contracts/security/index.md

```diff
@@ -90,7 +90,7 @@ After testing your contract, it is good to ask others to check the source code f
 
 Commissioning a smart contract audit is one way of conducting an independent code review. Auditors play an important role in ensuring that smart contracts are secure and free from quality defects and design errors.
 
-That said, you should avoid treating audits as a silver bullet. Smart contract audits won't catch every bug and are mostly designed to provide an additional round of reviews, which can help detect issues missed by developers during initial development and testing. You should also follow [best practices for working with auditors](https://twitter.com/tinchoabbate/status/1400170232904400897), such as documenting code properly and adding inline comments, to maximize the benefit of a smart contract audit.
+That said, you should avoid treating audits as a silver bullet. Smart contract audits won't catch every bug and are mostly designed to provide an additional round of reviews, which can help detect issues missed by developers during initial development and testing. You should also follow best practices for working with auditors (you can find insightful suggestions [here](https://twitter.com/tinchoabbate/status/1400170232904400897) and [here](https://inference.ag/blog/2023-08-14-tips/)), such as documenting code properly and adding inline comments, to maximize the benefit of a smart contract audit.
```

Comment:
We try to avoid single word links that don't describe the destination (ie. "here"). 

We could consider breaking these links out below the paragraph:

```suggestion
That said, you should avoid treating audits as a silver bullet. Smart contract audits won't catch every bug and are mostly designed to provide an additional round of reviews, which can help detect issues missed by developers during initial development and testing. You should also follow best practices for working with auditors, such as documenting code properly and adding inline comments, to maximize the benefit of a smart contract audit.

- [Smart contract auditing tips & tricks](https://twitter.com/tinchoabbate/status/1400170232904400897) - _@tinchoabbate_
- [Make the most out of your audit](https://inference.ag/blog/2023-08-14-tips/) - _Inference_
```